daisy_workflows/export/disk_export_ext.wf.json

```diff
@@ -32,37 +32,50 @@
     }
   },
   "Sources": {
-    "${NAME}_export_disk_ext.sh": "./export_disk_ext.sh"
+    "${NAME}_export_disk_ext.sh": "./export_disk_ext.sh",
+    "disk_resizing_mon.sh": "./disk_resizing_mon.sh"
   },
   "Steps": {
     "setup-disks": {
       "CreateDisks": [
         {
-          "Name": "disk-${NAME}",
+          "Name": "disk-${NAME}-os-${ID}",
```

Comment:
This disk doesnt need the id or exact name set as it is unused in the script (you just need this for the buffer disk)